module00/subject/en.subject.tex

```diff
@@ -1033,6 +1033,7 @@ \section*{Examples}
 	5552778
 \end{42console}
 
+\info{Take a look at the wonderfull tqdm library, it will come in handy in many situations}
```

Comment:
typo:
* wonderfull -> wonderful

Suggestion:
* tqdm -> \texttt{tqdm}

I am wondering about the formulation of this info buble, shall we rather write:
"Despite \texttt{tqdm} is forbidden, you can take a look at ..."